freqtrade/exchange/exchange.py

```diff
@@ -1174,6 +1177,28 @@ def get_rate(self, pair: str, refresh: bool, side: str) -> float:
 
         return rate
 
+    def get_rates(self, pair: str, refresh: bool) -> Tuple[float, float]:
+        buy_rate = sell_rate = None
+        if not refresh:
+            buy_rate, sell_rate = self._buy_rate_cache.get(pair), self._sell_rate_cache.get(pair)
+
+        bid_strategy = self._config.get('bid_strategy', {})
+        ask_strategy = self._config.get('ask_strategy', {})
+        order_book = ticker = None
+        if bid_strategy.get('use_order_book', False) and ('use_order_book' in bid_strategy):
```

Comment:
you can safely omit the 2nd part - it's not doing anything, really.

if any, you'd need to check if it's inside before access (exploiting condition shortcircuiting) - but that's not necessary when using `.get()`.

Reply:
updated